sdk/servicebus/azure-servicebus/azure/servicebus/_common/utils.py

```diff
@@ -330,3 +333,14 @@ def get_receive_links(messages):
     except AttributeError:
         pass
     return links
+
+
+def parse_sas_credential(credential):
+    # type: (AzureSasCredential) -> Tuple
+    sas = credential.signature
```

Comment:
Maybe `credential.signature.lower()` just to be safe?

Reply:
as far as I know, signature is case-sensitive -- if we call lower(), then the signature would become invalid.
is there anything else I'm unaware of?